app/controllers/admin/listing_shapes_controller.rb

```diff
@@ -10,28 +10,88 @@ class Admin::ListingShapesController < ApplicationController
     :name,
     :action_button_label,
     :shipping_enabled,
+    :price_enabled,
+    :online_payments,
     :units,
   ).with_validations {
     validates :name, presence: true
     validates :action_button_label, presence: true
     validates :shipping_enabled, inclusion: { in: [true, false] }
+    # TODO Add validations
   }
 
   def index
+    processes = TransactionService::API::Api.processes.get(community_id: @current_community.id)[:data]
+
     render("index",
            locals: {
              selected_left_navi_link: LISTING_SHAPES_NAVI_LINK,
+             templates: templates(processes),
              listing_shapes: all_shapes(@current_community.id)})
   end
 
+  def new
+    processes = TransactionService::API::Api.processes.get(community_id: @current_community.id)[:data]
```

Comment:
This should have a helper method that wraps calls to service the same way as we have get_shape, etc.